src/Library/demos/Button/main.js

```diff
@@ -0,0 +1,19 @@
+import Gtk from "gi://Gtk?version=4.0";
+import GObject from "gi://GObject";
+
+const test_button = workbench.builder.get_object("test_button");
+function handle_click() {
+  const dialog = new Gtk.MessageDialog({
+    text: "Button was Clicked",
+    transient_for: workbench.window,
+    modal: true,
+    buttons: Gtk.ButtonsType.OK,
+  });
+  dialog.connect("response", () => {
+    dialog.close();
+  });
+
+  dialog.present();
+}
+
+test_button.connect("clicked", handle_click);
```

Comment:
the dialog is a bit annoying while testing because the user needs to close the modal to proceed - it is 'blocking'

Also `Gtk.MessageDialog` will be deprecated in GTK 4.10

I don't have a much better idea - I'd say `console.log("Hello, World!");` is enough for now